libraries/GDBStub/src/GDBStub.h

```diff
@@ -1,6 +1,6 @@
 #ifndef GDBSTUB_H
```

Comment:
This #define is the same as the one in libraries/GDBStub/src/internal/gdbstub.h (file is near bottom of this diff). Is that on purpose? This can cause shadowing in the include chain.

Reply:
Removed this copy and moved the internal one into an easily accessed spot.  Only one [Gg][Dd][Bb]stub.h file now exists and it has all of @kylefleming's functions defined in it now.